astroplan/core.py

```diff
@@ -25,8 +40,10 @@ class Observer(object):
     Some comments.
     """
 
-    def __init__(self, name, longitude, latitude, elevation, pressure,
-                 relative_humidity, temperature, timezone, description=None):
+    def __init__(self, name=None, location=None, latitude=None, longitude=None,
```

Comment:
I think my preference is no to `kwargs` -- if the arguments are spelled out, it is easier to quickly see what is expected from the documentation of the call sequence.